prototype/core/annotation/src/main/java/org/eclipse/sensinact/core/annotation/verb/GET.java

```diff
@@ -73,6 +75,19 @@
      */
     Class<?> type() default Object.class;
 
+    /**
+     * Duration of the value cache. If the value is requested using a CACHED get and
+     * the cache duration is exceeded, the annotated method will be called.
+     *
+     * The default duration is in {@link TimeUnit#MILLISECONDS}.
+     */
+    long cacheDuration() default 500;
+
+    /**
+     * Unit of the value cache duration. Defaults to {@link ChronoUnit#MILLIS}.
+     */
+    ChronoUnit cacheDurationUnit() default ChronoUnit.MILLIS;
```

Comment:
Should this be TimeUnit rather than ChronoUnit? My reasoning is that we're dealing with relative times not dates, so TimeUnit is more appropriate. It also links with the JavaDoc for `cacheDuration` which says `TimeUnit`.

Reply:
As we use `Duration` internally, I'd prefer to keep using `ChronoUnit` and update the Javadoc